src/main/scala/chisel3/iotesters/VerilatorBackend.scala

```diff
@@ -333,16 +333,20 @@ private[iotesters] class VerilatorBackend(dut: Chisel.Module,
           (implicit logger: PrintStream, verbose: Boolean, base: Int): BigInt = {
     val idx = off map (x => s"[$x]") getOrElse ""
     val path = s"${signal.parentPathName}.${validName(signal.instanceName)}$idx"
-    val bigIntU = peek(path)
+    val bigIntU = simApiInterface.peek(path) getOrElse BigInt(rnd.nextInt)
```

Comment:
Ah to be more informative, this originally called peek(path: String blah blah ...) which returned a random number for reasons unknown to me.